scripts/gui/microbe_hud.mjs

```diff
@@ -481,27 +527,210 @@ function hideWinText(){
     document.getElementById("winContainer").style.display = "none";
 }
 
+// Variables needed for changes
+let barToChange = "";
+let titleToChange = "";
+let valuesToChange = "";
+let h = "";
+
+function onCompressPanelClicked() {
+
+    // If panel is already compress cannot compress again
+    const panelToChangeId = $(this).parent().parent().attr("id");
```

Comment:
I'm kinda feeling like this is not an optimal way to do things. The callbacks could be separated so that this trickery was not needed. Maybe something like:
```
    for (const element of compressPanels) {
        let theElementThisButtonControls = ...
        element.addEventListener("click",
            () => { onCompressPanelClicked(theElementThisButtonControls);}, true);
    }
```